### DISCUSSION TITLE
Could someone please tell me what buildType means in the document

### DISCUSSION THREAD

**Discussion by yuhang1995** [jaredpalmer/razzle#1896] [Q&A] [answered]
Could someone please tell me what buildType means in the [document](https://razzlejs.org/docs/customization#options)

I'm not sure about the difference between "iso" and "iso-serverless"

![image](https://user-images.githubusercontent.com/31569789/194691259-666474d3-24a4-43a3-a107-5ab7f3f3480b.png)

**Comment by fivethreeo** [comment, marked as answer]
"iso-serverless" bundles all node_modules serverside + client build,"iso" does not bundlenode_modules serverside + client build
